File: TollCalculator/src/main/java/se/kvrgic/Vehicle.java
package se.kvrgic;

public enum Vehicle {
    CAR("Car"),
    MOTORBIKE("Motorbike"),
    TRACTOR("Tractor"),
    EMERGENCY("Emergency"),
    DIPLOMAT("Diplomat"),
    FOREIGN("Foreign"),
    MILITARY("Military");
    private final String type;

    Vehicle(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }
}

File: C#/TollCalculator/Models/Vehicle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TollCalculator.Models
{
    public class Vehicle
    {
        public VehicleType vehicleType { get; set; }
        public bool IsTollFree()
        {
            return Enum.IsDefined(typeof(TollFreeVehicles), (int)this.vehicleType);
        }

    };

    public enum VehicleType
    {
        Motorbike = 0,
        Tractor = 1,
        Emergency = 2,
        Diplomat = 3,
        Foreign = 4,
        Military = 5,
        Car = 6,
    }
}